starter-code/js/index.js

```diff
@@ -1,52 +1,75 @@
-function deleteItem(e){
+function deleteItem(e) {
 
 }
 
-function getPriceByProduct(itemNode){
+function getPriceByProduct(itemNode) {
 
 }
 
-function updatePriceByProduct(productPrice, index){
+function updatePriceByProduct(productPrice, index) {
 
 }
 
+
+
 function getTotalPrice() {
+  var productArray = [document.getElementsByClassName('product')]
```

Comment:
```document.getElementsByClassName('product')``` already returns an array like structure. It returns an  HTMLCollection that has the .length property which you used below